src/Illuminate/Queue/RedisQueue.php

```diff
@@ -195,60 +198,43 @@ protected function migrate($queue)
     /**
      * Migrate the delayed jobs that are ready to the regular queue.
      *
-     * @param  string  $from
-     * @param  string  $to
+     * @param  string $from
+     * @param  string $to
```

Comment:
Just a PHPDoc nit: see https://laravel.com/docs/5.7/contributions#coding-style, should have 2 spaces in between.

There's a few spots as well.